src/games/strategy/engine/data/export/DefaultAttachmentExporter.java

```diff
@@ -138,12 +134,12 @@ private String printStringArrayOption(final Field field, final String option, fi
       if (valueArray == null) {
         return "";
       }
-      final Iterator<String> values = Arrays.asList(valueArray).iterator();
-      String value = values.next();
-      while (values.hasNext()) {
-        value = value + ":" + values.next();
+      StringBuilder valueBuilder = new StringBuilder();
+      for(String val : valueArray) {
+        valueBuilder.append(":").append(val);
```

Comment:
Same, can this be replaced with a Joiner? Ditto for the any other StringBuilders that we use in a loop.